xesmf/frontend.py

```diff
@@ -451,6 +451,17 @@ def __call__(self, indata, keep_attrs=False, skipna=False, na_thres=1.0):
             if `na_thres` is set to 1, all input values must be missing to
             mask the output value.
 
+        output_chunks: dict or tuple, optional
+            If indata is a dask_array_type, the desired chunks to have on the
+            output data along the spatial axes. Other non-spatial axes inherit
+            the same chunks as indata as those are not affected by the application
+            of the weights. Default behavior is to have the outdata chunks be like
+            the indata chunks. Chunks have to be specified for all spatial dimensions
+            of the output data otherwise regridding will fail. output_chunks can
+            either be a tuple the same size as the spatial axes of outdata or it
+            can be a dict with defined dims. If output_chunks is a dict, the
+            keys must match the dims of indata, e.g. {'lat':10,'lon':10}.
```

Comment:
```suggestion
            can be a dict with defined dims. If output_chunks is a dict, the
            keys must match the dims of the output grid passed when initializing this Regridder.
```
I think the code below is confusing the output and the input ? Seems to me that the output chunks should match the dimension names of `ds_out`, and not `indata` ?

But that raises a OOP issue : the `BaseRegridder` doesn't have a notion of output dimension names. This is instead stored in `Regridder.out_horiz_dims` and  `SpatialAverager.geom_dim_name`.  I see a few ways out:

1. Implement the dict -> tuple transformation as a method on each class.
2. Refactor so that `out_horiz_dims` is a property of the `BaseRegridder` (meaning a new `output_dims` arg should be added to its init).

I think n°2 is the cleanest result, but with the most (semi-breaking) changes.